### PR TITLE
Add Mintlify icon

### LINKED ISSUE #10934: Request: Mintlify
### Brand Name

Mintlify

### Website

https://mintlify.com/

### Popularity Metric

The similar web rank is 319,562. See https://www.similarweb.com/website/mintlify.com/

### Official Resources for Icon and Color

PNG Icon from GitHub: https://avatars.githubusercontent.com/u/93011474
Favicon on website: https://mintlify.com/favicon.ico
SVG from their documentation page: https://mintlify.s3-us-west-1.amazonaws.com/mintlify/logo/light.svg

### Additional Comments

The SVG has some letters in it, so we need to remove those.

### PR DESCRIPTION
**Pull request by deevee47**
**Issue:** closes #10934

**Popularity metric:**

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5,11 +5,14 @@
             "hex": "ECD53F",
             "source": "https://github.com/motdotla/dotenv/blob/40e75440337d1de2345dc8326d6108331f583fd8/dotenv.svg",
             "aliases": {
-                "aka": [
-                    "Dotenv"
-                ]
```
Comment (uncenter) [inline _data/simple-icons.json:10]: Don't make formatting changes please.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -5,11 +5,14 @@
...
+                "aka": ["Dotenv"]
             }
         },
+        {
+            "title": "Mintlify",
+            "hex": "18e299",
```
Comment (uncenter) [inline _data/simple-icons.json:13]:
```suggestion
            "hex": "18E299",
```

File: _data/simple-icons.json [hunk 3]
```diff
@@ -5,11 +5,14 @@
...
+            "source": "https://github.com/simple-icons/simple-icons/blob/develop/icons/mintlify.svg"
```
Comment (uncenter) [inline _data/simple-icons.json:14]: You can't link to our own repository for the source 😅

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (uncenter) [reply _data/simple-icons.json:10]: (Still lots of formatting changes...)

**Comment by adamrusted** [comment]
I've fixed the issues with the data JSON, @deevee47 - can you please make sure you are either using our prettier / eslint config, or disable your format-on-save functionality in your editor to prevent that happening again?

File: _data/simple-icons.json (hunk 3 above)
Comment (adamrusted) [reply _data/simple-icons.json]: I concur - you can use the homepage of the site as your source, as there's an SVG version of the icon there.

**Comment by uncenter** [comment]
It would also be nice if you could add a preview image of the icon using https://wasm.simpleicons.org/preview, as suggested in the comment of the pull request template.

**Comment by adamrusted** [comment]
Closing as abandoned. If you wish to continue working on this @deevee47 just let us know and we'll re-open.

**Pull request closed without merging**